test_infra/src/lib.rs

```diff
@@ -1037,6 +1037,34 @@ impl Guest {
         );
     }
 
+    pub fn check_vsock_guest_to_host(&self, socket: String) {
```

Comment:
`socket` can be `&str`, no?

Reply:
Yep, could keep the same signature. String is required because `socket` needs to be further passed into a thread, so would still need to convert later.

Thanks

Reply:
Done, thanks.